### ISSUE TITLE
add clarification to con hub alt-text to identify buttons

### ISSUE THREAD

**Issue by Raven-St** [ArisiaInc/a21-remote-site#418]
Based on accessibility feedback, it sounds like buttons on the con hub page are not being identified as buttons in the alt-text and should be.

**Comment by emtersoff** [comment]
Fixed.
